composer.json

```diff
@@ -10,7 +10,10 @@
     ],
     "require": {
         "php": "^7.1",
-        "phpstan/phpstan": "^0.10"
+        "phpspec/prophecy": "^1.7",
```

Comment:
As done elsewhere in similar extensions, this shouldn't require the packages directly. Also, they're already dev-required

Reply:
👍